### PR TITLE
DSOS-1987: allow specific tags for ec2 and ebs resources

### PR DESCRIPTION
**Pull request by drobinson-moj**
Finer control over tagging.  Allow additional tags specific to ec2_instance and ebs_volume resources.   This allows us to add a backup tag to only the ec2_instance resource, thus avoiding duplicate ec2 and ebs backups.  Tested in nomis accounts.

### CODE REVIEW

File: test/unit-test/data.tf [hunk 1]
```diff
@@ -21,6 +21,7 @@ data "aws_iam_policy_document" "ec2_test_policy" {
     #checkov:skip=CKV_AWS_109:
     #checkov:skip=CKV_AWS_111:
     #checkov:skip=CKV_AWS_107:
+    #checkov:skip=CKV_AWS_356:
```
Comment (ewastempel) [inline test/unit-test/data.tf:24]: Could you please add a description with a reason for skipping?

Comment (drobinson-moj) [reply test/unit-test/data.tf:24]: I was being consistent with the previous skips :). It's because the policy is just used within the unit-test only

Comment (ewastempel) [reply test/unit-test/data.tf:24]: I know, we've been lazy before :) but good to have that extra info going forward :)

**Review by ewastempel — APPROVED** [review]
LGTM

**Pull request merged**